release/ml_user_tests/horovod/compute_tpl.yaml

```diff
@@ -5,13 +5,13 @@ max_workers: 3
 
 head_node_type:
     name: head_node
-    instance_type: g3.8xlarge
+    instance_type: m5.4xlarge
```

Comment:
@amogkam it seems we don't need a GPU head node unless I am miscalculating.

We are running 6 workers. g3.8xlarge has 2 GPUs per node. So 3 workers alone should cut it?